README.org

```diff
@@ -76,7 +76,7 @@ Help is welcome!
 ** Building
 
 #+begin_example
-$ git clone git://github.com/djcb/mu.git
+$ git clone git@github.com:djcb/mu.git
```

Comment:
Think it should be `https://github.com/djcb/mu.git`, or? Fewer things can go wrong that way

Reply:
I tried to clone the repo, copy & paste from the readme, and it didnt work.
This was the input/output:
```
❯ git clone git://github.com/djcb/mu.git
Cloning into 'mu'...
fatal: unable to connect to github.com:
github.com[0: 140.82.121.4]: errno=Connection timed out
```
I just tried it again and gave me the same error.

Reply:
Yeah, `git://` won't work but `https://`:
```
git clone https://github.com/djcb/mu.git
Cloning into 'mu'...
remote: Enumerating objects: 37259, done.
```

Reply:
The link on the readme was not working.
I prefer ssh, but whatever works.
```suggestion
$ git clone https://github.com/djcb/mu.git
```